scripts/stale.py

```diff
@@ -35,32 +36,40 @@ def main():
     open_issues = repo.get_issues(state="open")
 
     for issue in open_issues:
+        comments = sorted([comment for comment in issue.get_comments()], key=lambda i: i.created_at, reverse=True)
+        last_comment = comments[0] if len(comments) > 0 else None
         if (
+            last_comment is not None and last_comment.user.login == "github-actions[bot]"
+            and not issue.assignees
+            and (dt.utcnow() - issue.updated_at).days > 7
+            and (dt.utcnow() - issue.created_at).days >= 30
+            and not any(label.name.lower() in LABELS_TO_EXEMPT for label in issue.get_labels())
+        ):
+            print(f"Would close issue {issue.number} since it has been 7 days of inactivity since bot mention.")
+            # issue.edit(state="closed")
+        elif (
             not issue.assignees
-            and (dt.utcnow() - issue.updated_at).days > 21
+            and (dt.utcnow() - issue.updated_at).days > 23
             and (dt.utcnow() - issue.created_at).days >= 30
             and not any(label.name.lower() in LABELS_TO_EXEMPT for label in issue.get_labels())
         ):
-            print("Closing", issue)
+            print(f"Would add stale comment to {issue.number}")
             # issue.create_comment(
-            #     "This issue has been automatically marked as stale and been closed because it has not had "
-            #     "recent activity. Thank you for your contributions.\n\nIf you think this still needs to be addressed"
-            #     " please comment on this thread."
+            #     "This issue has been automatically marked as stale because it has not had "
+            #     "recent activity. If you think this still needs to be addressed "
+            #     "please comment on this thread.\n\nPlease note that issues that do not follow the "
+            #     "[contributing guidelines](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md) "
+            #     "are likely to be ignored."
             # )
-            # issue.add_to_labels("wontfix")
-            # issue.edit(state="closed")
         elif (
             len(issue.assignees) > 0
             and (dt.utcnow() - issue.updated_at).days > 21
             and (dt.utcnow() - issue.created_at).days >= 30
+            and not any(label.name.lower() in LABELS_TO_EXEMPT for label in issue.get_labels())
         ):
             for assignee in issue.assignees:
-                print(f"Issue {issue.number}. Pinging {assignee.name} with message")
-                print(f"Hey @{assignee.login}, could you take a second look at this issue?")
-
-                # issue.create_comment(
-                #    f"Hey @{assignee.login}, could you take a second look at this issue?"
-                # )
+                print(f"This issue ({issue.number}) has been stale for a while, ping @{assignee.login}")
```

Comment:
Thank you, @LysandreJik 
I only have a comment about:

> If there are assignees on the issue/PR, then it puts the following comment: f"This issue has been stale for a while, ping @{assignee.login}"

This creates unnecessary noise. e.g. one Issue that had 2 assignees received this comment twice for each assignee and we were notified twice as well on our own and each other's warning.

What I'm proposing is:
1. there should be only one comment issued regardless of how many assignees are 
2. To solve (1) easier I am not sure there is a need to mention the assignee at all - i.e. no need to make a special case for Issues with assignees - whatever the general message is sent is good enough. The assignee gets it anyway - I don't think an assignee can unsubscribe (I could be wrong), so the tagging doesn't add anything since the person is already subscribed.

Bottom line - if the subscribed person with the right perms wants to do something about the Issue/PR they can. If they don't tagging won't do anything useful OMHO.

Reply:
Let's go with 2a! Thanks for your comment